src/Integration.UnitTests/ProfileConflicts/RuleSetInspectorTests.cs

```diff
@@ -372,10 +396,12 @@ public void RuleSetInspector_FixConflictingRules_NoConflicts()
             AssertNoConflictsExpected(solutionRuleSet.FilePath, projectRuleSet.FilePath);
 
             // Act
-            RuleSet target = this.testSubject.FixConflictingRules(solutionRuleSet.FilePath, projectRuleSet.FilePath);
+            FixedRuleSetInfo fixedInfo = this.testSubject.FixConflictingRules(solutionRuleSet.FilePath, projectRuleSet.FilePath);
```

Comment:
I had to return reach information about the fix so modified the tests to deal with that API change (and test it).